Monika After Story/game/dev/dev_farewells.rpy

```diff
@@ -0,0 +1,53 @@
+# dev related farewells
+
+init 5 python:
+    rules = dict()
+    rules.update(MASNumericalRepeatRule.create_rule(repeat=EV_NUM_RULE_YEAR))
+    addEvent(
+        Event(
+            persistent.farewell_database,
+            eventlabel="bye_st_patrick",
+            start_date=datetime.datetime(2017, 3, 17, 5),
+            end_date=datetime.datetime(2017, 3, 18, 5),
+            unlocked=True,
+            rules=rules
+        ),
+        eventdb=evhand.farewell_database
+    )
+    del rules
+
+label bye_st_patrick:
+    m 1c "Aww, leaving already?"
+    m 1e "It's really sad whenever you have to go..."
+    m 1b "Good luck with the hangover!"
+    return 'quit'
+
+init 5 python:
+    ev = Event(persistent.farewell_database,eventlabel="bye_dev",unlocked=True)
```

Comment:
While this works, this bypasses the init locks system. Doing the traditional:
```python
<setup rules>
addEvent(
    Event(
         ....
         rules=rules
     )
)
del rules
```
allows for the initlocks system to disable changes on init.

Since this is a dev goodbye, its not really an issue, but in production, we should anonymously create Event instances as we call addEvent.
**No change here is necessary, just letting you know of  a standard**

(If you already knew this, then dont mind me~)

Reply:
Actually I didn't knew that, I did it that way just because I got curious about the the optional argument on the rules and wanted to do a little test with that, that's why I did it on the dev one.
Thanks for explaining that, I'll be sure to never do it this way in any production greeting/farewell.